manifests/api_service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  name: v1alpha1.token.kubevirt.io
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
```

Comment:
Will it break deployments on plain Kubernetes?

Reply:
Currently the proxy does not work on plain k8s, because of this annotation and a similar one in `serivce.yaml`.
They make the implementation simpler. There is already an open issue https://github.com/kubevirt/vm-console-proxy/issues/16 to do this certificate generation manually and to not depend on openshift.